### ISSUE TITLE
Desktop: Accessibility: Toggling the notebook list or note list with F10 or F11 should announce the change for screen reader users

### ISSUE THREAD

**Issue by personalizedrefrigerator** [laurent22/joplin#11741]
See [this feedback from the forum](https://discourse.joplinapp.org/t/calling-visually-impaired-users-help-us-improve-accessibility/42094/7?u=personalizedrefriger).

**Comment by personalizedrefrigerator** [comment]
Still an issue!
